utils/hare-status

```diff
@@ -177,9 +177,20 @@ def processes(cns: Consul, consul_util: ConsulUtil,
     node_health = consul_util.get_node_health_status(node_name)
     process_states = get_node_process_states(cns, node_name)
     process_list = get_node_processes(cns, node_name)
+
```

Comment:
Do we need to have decorator `@repeat_if_fails` for this function processes()?

Reply:
@suvratjoshi Above function is getting called from functions(**show_text_status** and **get_cluster_status**) and we have '@repeat_if_fails' for both of them